DOC: document actions (#16)
pyansys/actions: README.rst

   :target: https://opensource.org/licenses/MIT
   :alt: MIT

A repository containing a collection of `GitHub Workflows
<https://docs.github.com/en/actions/using-workflows/about-workflows>`_ to be
reused by projects in the PyAnsys ecosystem.

> Refer to `actions.docs.pyansys.com <https://actions.docs.pyansys.com>`_ for more
> information about available actions and how to use those.

Reviewer: ```suggestion
For more information on available actions and how to use them, see `actions.docs.pyansys.com <https://actions.docs.pyansys.com>`_ .
```
Style guidelines say to start with why they would go see a resource and to use "on" as the preposition.

Author: Taking notes on this, thanks for remembering this, Kathy.
